challenges/objects-arrays.js

```diff
@@ -108,20 +168,31 @@ The zoos are concerned about animals with a lower population count. Using filter
 
 */
 const lowPopulationAnimals = [];
+const animals = zooAnimals;
+animals.filter((zooAnimals) => {
+	zooAnimals.population === animals;
+});
 console.log(lowPopulationAnimals);
 
```

Comment:
No array is actually being filtered, and think about your comparison here, are you comparing what you want to? -2